docs/MODDING.md

```diff
@@ -188,4 +188,4 @@ Assuming all went well, you can now push your changes to your fork with the foll
 
 ```bash
 git push origin <YOUR BRANCH NAME>
-```
\ No newline at end of file
+```
```

Comment:
anyone know what this change is? it doesn't appear to be newline/not

Reply:
Well it is, but github apparently displays it differently for markdown files than it does for code files.  I checked out this branch and opened the file up in a hex editor just to confirm:  
![image](https://github.com/HarbourMasters/Shipwright/assets/30329717/571eff9d-ba99-4b43-9e40-d0fc9cf060e8)

Reply:
Since the other markdown files end in newlines, it should be fine to leave it.